DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/Index.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@using Microsoft.Extensions.Configuration
+@model AllDelegatesViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/allDelegates.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Delegates";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Delegates/All";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="~/Views/TrackingSystem/Delegates/Shared/_DelegatesSideNavMenu.cshtml" model="DelegatePage.AllDelegates" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-one-third">
+        <h1 id="page-heading" class="nhsuk-heading-xl">Delegates</h1>
+      </div>
+      <div class="nhsuk-grid-column-two-thirds all-delegates-button-group">
+        <button class="nhsuk-button nhsuk-button--secondary all-delegates-button">
+          Register
+        </button>
+        <button class="nhsuk-button nhsuk-button--secondary all-delegates-button">
+          Email
+        </button>
+        <button class="nhsuk-button nhsuk-button--secondary all-delegates-button">
+          Upload
+        </button>
+        <button class="nhsuk-button nhsuk-button--secondary all-delegates-button">
+          Download
+        </button>
+      </div>

```

Comment:
Does this collapse in a sensible way for mobile?

Reply:
Screenshots attached.